src/packages/protoc-gen-openapiv2/update-hash.py

```
@@ -0,0 +1,166 @@
+            return f"sha256-{hash_match.group(1)}"
+        
+        # Alternative pattern
+        hash_match = re.search(r'expected:\s*sha256-([A-Za-z0-9+/=]+)', error_output)
+        if hash_match:
+            return f"sha256-{hash_match.group(1)}"
+            
+        print("Could not extract vendor hash from build output")
+        print("Build stderr:", error_output)
+        return None
+        
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)